sympy/parsing/tests/test_c_parser.py

```diff
@@ -5089,13 +5096,13 @@ def test_while_stmt():
         c_src4 = (
             'int digit_sum(int n)'+
             '{' + '\n' +
-                'int sum = 0;' + '\n' +
+                'int ans = 0;' + '\n' +
                 'while(n > 0)' + '\n' +
                 '{' + '\n' +
-                    'sum += (n % 10);' + '\n' +
+                    'ans += (n % 10);' + '\n' +
                     'n /= 10;' + '\n' +
                 '}' + '\n' +
-                'return sum;' + '\n' +
+                'return ans;' + '\n' +
```

Comment:
I don't think that the original test cases should be changed without reason, if the test cases are going to be `xfail`d

Reply:
I changed the variable name because `sum` is a built-in function in Python, and this was causing a `SymPyDeprecationWarning` to show up:
```
/home/tirthankar/sympy/sympy/codegen/ast.py:211: SymPyDeprecationWarning:

  The string fallback in sympify() is deprecated.

  To explicitly convert the string form of an object, use
  sympify(str(obj)). To add define sympify behavior on custom
  objects, use sympy.core.sympify.converter or define obj._sympy_
  (see the sympify() docstring).

  sympify() performed the string fallback resulting in the following string:

  '<built-in function sum>'

  See https://docs.sympy.org/latest/explanation/active-deprecations.html#deprecated-sympify-string-fallback
  for details.

  This has been deprecated since SymPy version 1.6. It
  will be removed in a future version of SymPy.

    return cls._get_constructor(attr)(arg)
```
Changing the variable name prevents this deprecation warning from showing up, so I don't think the change is unwarranted.

Reply:
That is weird because that is valid C code
and users shouldn't be responsible for naming conflicts of C variables with python builtins. 

I think that the tests should be kept as is now because we can rather catch something suspicious happening with sympify there.

Reply:
Okay, I'll revert this change.